bots/pr/src/main/java/org/openjdk/skara/bots/pr/JEPCommand.java

```diff
@@ -55,23 +55,39 @@ private void showHelp(PrintWriter reply) {
                  * `/jep 1234567`
                  * `/jep jep-123`
                  * `/jep JEP-123`
+                 * `/jep 123`
                  * `/jep unneeded`
 
                 Note:
-                The project prefix (`JDK-` in the above examples) is optional if you use an issue ID.
-                The issue type in that case must be `JEP`.
-                The `JEP-` or `jep-` prefix is required if you instead provide a JEP ID.
+                The prefix (`JDK-`, `JEP-` or `jep-` in the above examples) is optional.
+                The bot firstly treats the ID without prefix as a JEP ID.
+                If not found, the bot then treats the ID without prefix as an issue ID.
+                The issue type in any case must be `JEP`.
                 """);
     }
 
-    private Optional<Issue> getJepIssue(String args, PullRequestBot bot) {
-        Optional<Issue> jbsIssue;
-        if (args.startsWith("jep-") || args.startsWith("JEP-") || args.startsWith("Jep-")) {
-            // Handle the JEP ID
+    private Optional<Issue> getJepIssue(String args, PullRequestBot bot, PrintWriter reply) {
+        Optional<Issue> jbsIssue = Optional.empty();
+        var upperArgs = args.toUpperCase();
+        if (upperArgs.startsWith("JEP-")) {
+            // Handle the JEP ID with `JEP` prefix
             jbsIssue = bot.issueProject().jepIssue(args.substring(4));
         } else {
-            // Handle the issue ID
-            jbsIssue = bot.issueProject().issue(args);
+            if (!upperArgs.startsWith(bot.issueProject().name().toUpperCase())) {
+                // Handle the raw JEP ID without `JEP` prefix and project prefix. If the JEP has the same ID
+                // as any issue, the bot firstly parse the ID as JEP instead of general issue.
+                // For example, if we have a `JEP-12345` (its issue ID is not `JDK-12345`) and an issue `JDK-12345`,
+                // when typing `/jep 12345`, the bot firstly parses it as `JEP-12345` instead of `JDK-12345`.
+                jbsIssue = bot.issueProject().jepIssue(args);
+                if (jbsIssue.isEmpty()) {
+                    reply.println("The JEP issue of the JEP argument `" + args + "` was not found. " +
```

Comment:
Do we really need to print a message about this?

Reply:
I also noted that on line 150 and 156, the message doesn't start with a capital letter. Could you fix that too?

Reply:
I think that will be clear enough without this extra message.

Reply:
Fixed.